impeller/typographer/rectangle_packer.cc

```diff
@@ -50,10 +51,14 @@ class SkylineRectanglePacker final : public RectanglePacker {
   // the skyline segments >= 'skylineIndex'? If so, return true and fill in
   // 'y' with the y-location at which it fits (the x location is pulled from
   // 'skylineIndex's segment.
-  bool rectangleFits(int skylineIndex, int width, int height, int* y) const;
+  bool rectangleFits(size_t skylineIndex, int width, int height, int* y) const;
```

Comment:
Here and elsewhere, use the `PascalCase` per the style guide?

Reply:
Fixed